modules/management/default.nix

```diff
@@ -1,6 +1,4 @@
 {
   imports = [
-    ./bookstack.nix
```

Comment:
Don't kill our book stack

Reply:
why is it included there?

Reply:
Because bookstack is migrated into its own vm yet.